plugins/iterm2/iterm2.plugin.zsh

```diff
@@ -0,0 +1,38 @@
+#####################################################
+# iTerm2 plugin for oh-my-zsh                       #
+# Author: Aviv Rosenberg (github.com/avivrosenberg) #
+#####################################################
+
+###
+# This plugin is only relevant if the terminal is iTerm2 on OSX.
+if [[ "$OSTYPE" == darwin* ]] && [[ -n "$ITERM_SESSION_ID" ]] ; then
+
+  ###
+  # iterm2_profile(): Function for changing the current terminal window's
+  # profile (colors, fonts, settings, etc).
+  # To change the current iTerm2 profile, call this function and pass in a name
+  # of another existing iTerm2 profile (name can contain spaces).
+  function iterm2_profile() {
+    # Desired name of profile
+    local profile="$1"
+
+    # iTerm2 escape code for changing profile
+    local iterm2_code="\x1b]50;SetProfile=$profile\x7"
```

Comment:
Why hex escapes here and octal escapes below? Might be more readable with octal throughout.